v2/datastream-to-spanner/pom.xml

```diff
@@ -48,6 +48,11 @@
             <artifactId>json</artifactId>
             <version>${json.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.cloud</groupId>
+            <artifactId>google-cloud-spanner</artifactId>
+            <version>6.62.0</version>
```

Comment:
How was it working earlier without this dependency?

Reply:
IIUC you'll have to modify https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/1df7e0ae278f214fea87a20731e525875f4055c7/v2/spanner-common/src/main/java/com/google/cloud/teleport/v2/spanner/type/Type.java#L41.

Reply:
> How was it working earlier without this dependency?

The dependency was indirectly coming from Apache Beam. But, now since you have to override the client library version, this is needed.

Reply:
@arawind I have already modified the `Type.java` file and as @darshan-sj mentioned, the dependency was added to bypass the beam.